src/plugins/kibana_react/public/page_template/solution_nav/solution_nav_avatar.tsx

```diff
@@ -27,13 +32,30 @@ export type KibanaPageTemplateSolutionNavAvatarProps = DistributiveOmit<EuiAvata
 export const KibanaPageTemplateSolutionNavAvatar: FunctionComponent<
   KibanaPageTemplateSolutionNavAvatarProps
 > = ({ className, size, ...rest }) => {
+  const { euiTheme } = useEuiTheme();
+
+  const pageTemplateSolutionNavAvatarStyles = {
+    base: css`
+      ${css(useEuiShadow('s'))};
+    `,
+    xxl: css`
+      ${useEuiShadow('m')};
+      line-height: 100px;
+      border-radius: 100px;
+      display: inline-block;
+      background: ${euiTheme.colors.backgroundBasePlain} url('../../assets/texture.svg') no-repeat;
+      background-size: cover, 125%;
+      text-align: center;
+    `,
+  };
+
   return (
-    // @ts-ignore Complains about ExclusiveUnion between `iconSize` and `iconType`, but works fine
+    // @ts-expect-error Complains about ExclusiveUnion between `iconSize` and `iconType`, but works fine
     <EuiAvatar
       className={classNames(
-        'kbnPageTemplateSolutionNavAvatar',
+        pageTemplateSolutionNavAvatarStyles.base,
         {
-          [`kbnPageTemplateSolutionNavAvatar--${size}`]: size,
+          [pageTemplateSolutionNavAvatarStyles.xxl]: size === 'xxl',
```

Comment:
I'm not entirely sure we're not able to leverage `@emotion/react` and set the `css` property on the `EuiAvatar`.

Reply:
I'm wondering the same. Is there a reason?
It should just be something like:

```ts
css={[pageTemplateSolutionNavAvatarStyles.base, size === 'xxl' && pageTemplateSolutionNavAvatarStyles.xxl]}
```

Reply:
I tried a few variations, including removing the `className` prop altogether, but unfortunately I haven't found the magical incantation needed. I wonder if it's a Babel configuration thing.

Reply:
The `css` prop needs to be supported via [babel preset](https://emotion.sh/docs/css-prop#babel-preset), yes. I thought this is done for Kibana globally, but maybe not? 🤔